__tests__/app.test.js

```diff
@@ -32,4 +32,38 @@ describe('1 GET/api/topics', () => {
             expect(body.msg).toBe('route does not exist')
         })
     });
+});
+describe('2. GET API ARTICLES', () => {
+    test('status 200, should respond with an articles object with an array of articles', () => {
+        return request(app).get("/api/articles").expect(200).then((response)=>{
+            expect(response.body.articles).toEqual(expect.any(Array))
```

Comment:
This assertion is not required because your next 2 assertions do so implicitly. You check it has a length and therefore is not an empty array which is crucial and then you use the forEach which is an array method - proving it is an array.